<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XRtmpStreamerClass</class>
 <widget class="QWidget" name="XRtmpStreamerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1388</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XRtmpStreamer</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>推流RTMP   URL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>输入URL or 0/1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>美颜级别：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbLevel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>89</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>一级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>二级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三级</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="inUrl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="rtmpUrl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="XRtmpStreamer.qrc"/>
 </resources>
 <connections/>
</ui>
